### PR TITLE
media-video/praxis-live-bin: add 3.1.0

### PR DESCRIPTION
**Pull request by mrVanDalo**
License information is requested and will be added as soon it is available.

### CODE REVIEW

File: media-video/praxis-live/praxis-live-3.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	fperms +x /usr/lib/praxis-live/bin/praxis
+	fperms +x /usr/lib/praxis-live/bin/praxis_live
+
+	sed -i \
+	-e 's/Icon=.*/Icon=praxis_live/' \
+	-e 's/Categories=.*/Categories=AudioVideo;Audio;AudioVideoEditing/' \
+	usr/share/applications/praxis_live.desktop || die 'sed on desktop file failed'
+	domenu usr/share/applications/praxis_live.desktop
+
+	# todo make jdkhome use $JAVA_HOME
```
Comment (simonvanderveldt) [inline media-video/praxis-live/praxis-live-3.1.0.ebuild]: This seems to be fixed by the patch, right?

Comment (mrVanDalo) [inline media-video/praxis-live/praxis-live-3.1.0.ebuild]: yeah true

**Comment by simonvanderveldt** [comment]
We should probably drop the lintian directory, afaik it's Debian specific
```
$ ls -ahl /var/tmp/portage/media-video/praxis-live-3.1.0/image/usr/share/lintian/total 26K
drwxr-xr-x 3 root root 3 Jun  7 20:27 .
drwxr-xr-x 6 root root 6 Jun  7 20:27 ..
drwxr-xr-x 2 root root 3 Jun  7 20:27 overrides
$ ls -ahl /var/tmp/portage/media-video/praxis-live-3.1.0/image/usr/share/lintian/overrides/
total 22K
drwxr-xr-x 2 root root   3 Jun  7 20:27 .
drwxr-xr-x 3 root root   3 Jun  7 20:27 ..
-rw-r--r-- 1 root root 191 Jun  7 20:27 praxis-live
$ cat /var/tmp/portage/media-video/praxis-live-3.1.0/image/usr/share/lintian/overrides/praxis-live 
praxis-live: arch-independent-package-contains-binary-or-object
praxis-live: missing-dependency-on-libc
praxis-live: debian-changelog-file-missing
praxis-live: extended-description-is-empty
```

There are windows executables being installed, from a `.deb`, wtf !?
```
$ ls -ahl /var/tmp/portage/media-video/praxis-live-3.1.0/image/usr/lib/praxis-live/bin/
total 3.6M
drwxr-xr-x  2 root root    8 Jun  7 20:27 .
drwxr-xr-x 10 root root   10 Jun  7 20:27 ..
-rwxr-xr-x  1 root root 2.5K Jun  7 20:27 praxis
-rw-r--r--  1 root root 1.3M Jun  7 20:27 praxis64.exe
-rw-r--r--  1 root root 348K Jun  7 20:27 praxis.exe
-rwxr-xr-x  1 root root 2.5K Jun  7 20:27 praxis_live
-rw-r--r--  1 root root 1.3M Jun  7 20:27 praxis_live64.exe
-rw-r--r--  1 root root 348K Jun  7 20:27 praxis_live.exe
$ file /var/tmp/portage/media-video/praxis-live-3.1.0/image/usr/lib/praxis-live/bin/praxis64.exe 
/var/tmp/portage/media-video/praxis-live-3.1.0/image/usr/lib/praxis-live/bin/praxis64.exe: PE32+ executable (GUI) x86-64, for MS Windows
```

Also, I believe symlinks like this are normally relative?
I.e. it would be `../lib/praxis-live/bin/praxis`
```
$ ls -ahl /var/tmp/portage/media-video/praxis-live-3.1.0/image/usr/bin/
total 18K
drwxr-xr-x 2 root root  4 Jun  7 20:27 .
drwxr-xr-x 5 root root  5 Jun  7 20:27 ..
lrwxrwxrwx 1 root root 31 Jun  7 20:27 praxis -> /usr/lib/praxis-live/bin/praxis
lrwxrwxrwx 1 root root 36 Jun  7 20:27 praxis_live -> /usr/lib/praxis-live/bin/praxis_live
```

**Comment by mrVanDalo** [comment]
right, I forgot about them, I was creating the build on one day and forgot about that stuff on the next day. 

I'm on it.

**Comment by trebmuh** [comment]
> There are windows executables being installed, from a .deb, wtf !?

It can be done.

That said, this .deb package doesn't look to come **from** debian: 
https://packages.debian.org/praxis

**Comment by mrVanDalo** [comment]
Thats right, the .deb comes from github. From the release page of the project.

**Comment by mrVanDalo** [comment]
It is running on a system Java, but it needs a jdk.

**Pull request merged**